components/TanstackTable/Pagination.tsx

```diff
@@ -0,0 +1,43 @@
+'use client'
+
+import { ChevronLeftIcon, ChevronRightIcon } from '@radix-ui/react-icons'
+import { Button, IconButton } from '@radix-ui/themes'
+import React from 'react'
+import { Pagination as HeadlessPagination } from 'react-headless-pagination'
+import styles from './pagination.module.css'
+
+
+interface PaginationProps {
+  page: number
+  setPage: (page: number) => void
+  totalPages: number
+}
+
+const Pagination = ({ page, setPage, totalPages }: PaginationProps) => {
```

Comment:
```suggestion
const Pagination: FC<PaginationProps> = ({ page, setPage, totalPages }) => {
```

Reply:
Resolved in [5c5505a](https://github.com/imperial/ug-admissions/pull/2/commits/5c5505a10711d439db85b5816413685733d6c0b8)